com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -222,6 +268,34 @@ internal NetworkSceneManager(NetworkManager networkManager)
             ScenesLoaded.Add(DontDestroyOnLoadScene.handle, DontDestroyOnLoadScene);
         }
 
+        /// <summary>
+        /// If the VerifySceneBeforeLoading delegate handler has been set by the user, this will provides
+        /// an additional level of security and/or validation that the scene being loaded in the specified
+        /// loading mode is "a valid scene to be loaded in the LoadSceneMode specified".
+        /// </summary>
+        /// <param name="sceneIndex">Build Settings Scenes in Build List index of the scene</param>
+        /// <param name="sceneName">Name of the scene</param>
+        /// <param name="loadSceneMode">LoadSceneMode the scene is going to be loaded</param>
+        /// <returns>true (Valid) or false (Invalid)</returns>
+        internal bool ValidateSceneBeforeLoading(uint sceneIndex, string sceneName, LoadSceneMode loadSceneMode)
+        {
+            var validated = true;
```

Comment:
Wouldn't you just pass in the sceneIndex and then look up the scene name from `ScenesInBuild`?